modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/config/InMemoryConfigManager.java

```diff
@@ -22,14 +22,19 @@
 import java.util.Map;
 
 /**
- * In-memory Siddhi Configuration Manager
+ * In-memory Siddhi Configuration Manager.
  */
 public class InMemoryConfigManager implements ConfigManager {
-    private Map<String, String> masterConfigs = new HashMap<>();
+    private Map<String, String> extensionMasterConfigs = new HashMap<>();
+    private Map<String, Map<String, String>> storeConfigs = new HashMap<>();
```

Comment:
Fixed in 499aa1a.

Changed the variable to hold a map of properties (Map key should start with the store name). extractStoreConfigs will use name as a prefix to return individual store configs.